webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -1532,9 +1538,9 @@
 
 
 ### ###########################################
-### beol:letter
+### beol:basicLetter
 
-:letter rdf:type owl:Class ;
+:basicLetter rdf:type owl:Class ;
```

Comment:
I like `basicLetter`, but I'm wondering if it really belongs in the `beol` ontology. Maybe it would make sense to put it in the `biblio` ontology?

Reply:
no, in my opinion, it should stay in beol ontology because in Biblio Ontology we just have the items which appear in a bibliography list. On the other hand, changing this will cause a loop in loading ontologies. On the other hand, we then would need to immensely change the structure of data and letter templates

Reply:
OK, no problem.